### PR TITLE
Sidecar and Netflix are now working fine!

### PR DESCRIPTION
**Pull request by ZISQO**
using shikigva value

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ This repository is **explicitly** designed for Aorus Z390 Pro i7-9700K RX580 to
 
 - **Safari Netflix won't work with my RX580**, but it might work on navi GPUs (e.g. RX5700XT)
 - **Sidecar** and **HEVC** seemed boken on Catalina 10.15.3
+- **Using boot-arg shikigva value and Set to Yes Automatic Platform will Netflix in Safari and Sidecar worked
```
Comment (sarkrui) [inline README.md:37]: Perhaps you could rephrase the sentence a bit. 
Using boot-arg `shikigva=80` and Set `Automatic Platform` (iMac19,1 SMBIOS)  to `Yes` will allow Netflix (DRM contents) to be played in Safari.

**Review by sarkrui — COMMENTED** [review]
A bit of change would be better! thank you for the PR!

File: README.md (hunk 1 above)
Comment (ZISQO) [reply README.md:37]: sure~ : )

**Comment by sarkrui** [comment]
Thanks!

**Pull request closed without merging**